src/libsystemd-network/ndisc-protocol.c

```diff
@@ -0,0 +1,35 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include "ndisc-protocol.h"
+
+static const uint8_t prefix_length_code_to_prefix_length[_PREFIX_LENGTH_CODE_MAX] = {
+        [PREFIX_LENGTH_CODE_96] = 96,
+        [PREFIX_LENGTH_CODE_64] = 64,
+        [PREFIX_LENGTH_CODE_56] = 56,
+        [PREFIX_LENGTH_CODE_48] = 48,
+        [PREFIX_LENGTH_CODE_40] = 40,
+        [PREFIX_LENGTH_CODE_32] = 32,
+};
+
+int pref64_plc_to_prefix_length(uint16_t plc, uint8_t *ret) {
+        assert(ret);
```

Comment:
So, now the restriction to `ret` should be dropped, as on verify, it is not necessary to store the result.

Reply:
```sd_ndisc_router_prefix64_get_prefixlen``` need it

Reply:
I mean, please drop the assertion.
```c
if (ret)
        *ret = ...;
```

Reply:
OK